test/e2e/portforward.go

```diff
@@ -153,13 +161,25 @@ var _ = Describe("Port forwarding", func() {
 			logOutput := runKubectlWithTimeout(wait.ForeverTestTimeout, "logs", fmt.Sprintf("--namespace=%v", framework.Namespace.Name), "-f", podName)
 			verifyLogMessage(logOutput, "Accepted client connection")
 			verifyLogMessage(logOutput, "Expected to read 3 bytes from client, but got 0 instead")
+			poderr = framework.Client.Pods(framework.Namespace.Name).Delete(pod.Name,
```

Comment:
The pod deletions aren't strictly required; `framework` has an `AfterEach` that deletes the namespace.